mobile_config/tests/integrations/gateway_tracker.rs

```diff
@@ -1,53 +1,124 @@
-use crate::common::{
-    gateway_metadata_db::{create_tables, insert_gateway},
-    make_keypair,
-};
+use crate::common::{gateway_metadata_db, make_keypair};
 use chrono::{Timelike, Utc};
 use custom_tracing::Settings;
 use mobile_config::gateway::{
     db::{Gateway, GatewayType},
     tracker,
 };
+use rand::{seq::SliceRandom, thread_rng};
 use sqlx::PgPool;
 
 #[sqlx::test]
 async fn execute_test(pool: PgPool) -> anyhow::Result<()> {
```

Comment:
Can we get a more descriptive name for this test?

Is the difference between inserting `100` vs `1,000` vs `10,000` gateways in this consequential?
10k takes about 5s on my machine, if the volume isn't what we're after do we need to eat that time?